administration/src/util/translations/de.json

```diff
@@ -441,7 +441,12 @@
     "region": "Region",
     "cardsCreated": "Erstellte Karten",
     "cardsActivated": "Davon aktiviert",
-    "exportCsvNotPossible": "Etwas ist schiefgegangen beim Export der CSV."
+    "cardsActivatedBlue": "Blaue Ehrenamtskarte (aktiviert)",
+    "cardsActivatedGolden": "Goldene Ehrenamtskarte (aktiviert)",
+    "exportCsvNotPossible": "Etwas ist schiefgegangen beim Export der CSV.",
+    "noDataRegionInfo": "Für diese Region sind keine Daten vorhanden",
```

Comment:
"Keine Daten für den ausgewählten Zeitraum" ?
because there may be data “for this region” but it is not displayed because of the date filter